lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -60,6 +69,48 @@ const char *location_service_get_certificate_nrf_cloud(void)
 	return tls_certificate;
 }
 
+#if defined(CONFIG_NRF_CLOUD_MQTT)
+static void location_service_location_ready_cb(const struct nrf_cloud_cell_pos_result *const result)
+{
+	if (result != NULL) {
+		nrf_cloud_location.latitude = result->lat;
+		nrf_cloud_location.longitude = result->lon;
+		nrf_cloud_location.accuracy = (double)result->unc;
+	}
+	k_sem_give(&location_ready);
+}
+
+int location_service_get_cell_location_nrf_cloud(
+	const struct lte_lc_cells_info *cell_data,
+	char * const rcv_buf,
+	const size_t rcv_buf_len,
+	struct multicell_location *const location)
+{
+	ARG_UNUSED(rcv_buf);
+	ARG_UNUSED(rcv_buf_len);
+
+	int err;
+
+	err = nrf_cloud_cell_pos_request(cell_data, true, location_service_location_ready_cb);
```

Comment:
seeing this function on its own without any calls to `nrf_cloud_init` or `nrf_cloud_connect` let me to create https://github.com/nrfconnect/sdk-nrf/pull/6176

Reply:
We had some discussion if we could check if MQTT connection is up but looking at your PR, this is likely not possible.

Reply:
Great, thanks! I added a new error print out here in case nrf_cloud_cell_pos_request() returns -EACCES.